Hestia/app/src/main/java/hestia/backend/exceptions/ServerExceptions/ServerException.java

```diff
@@ -0,0 +1,13 @@
+package hestia.backend.exceptions.ServerExceptions;
+
+public abstract class ServerException extends Exception {
```

Comment:
oh my bad, must have been an error while importing these classes, definedServerException should indeed have indeed inherrited from serverExeption, will fix this